nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -846,9 +848,9 @@ def _transcribe_forward(
         """
         if isinstance(batch, PromptedAudioToTextMiniBatch):
             # Handling regular Canary DataLoader
-            audio = batch.audio
-            audio_lens = batch.audio_lens
-            decoder_input_ids = batch.prompt
+            audio = batch.audio.to(trcfg._internal.device)
+            audio_lens = batch.audio_lens.to(trcfg._internal.device)
+            decoder_input_ids = batch.prompt.to(trcfg._internal.device)
```

Comment:
Is this cast really needed? I thought PTL takes care of that.

Reply:
I saw a bug, but can't reproduce it now. Removed it